internal/pkg/cli/cmd/cmd.go

```diff
@@ -26,6 +28,34 @@ import (
 	versionCheck "github.com/keboola/keboola-as-code/pkg/lib/operation/remote/version/check"
 )
 
+// nolint: gochecknoinits
+func init() {
```

Comment:
Init sa vola pri inicializacii balicka.
Nemal by sa velmi pouzivat.
No kniznica `cobra` ma niektore nastavenia ako `singleton`, a preto to musim spravit takto (inak by vznikala data-race pri paralelnych testoch).